eth/api_backend.go

```diff
@@ -330,6 +330,10 @@ func (b *EthAPIBackend) BundlePrice() *big.Int {
 	return bundles[idx].Price
 }
 
+func (b *EthAPIBackend) Bundles(_ context.Context, fromBlock, toBlock int64) map[int64][]*types.Bundle {
```

Comment:
`map` is not suggested here.
I will suggest use Slice, and even one hight do not have any bundle, we should add an empty item there.

`Bundle` should not have the detailed tx, otherwise builder will leak their private tx, which is not expected.

Reply:
Also, need check the `toBlock`, if it comes from future, we should let the API caller knows, we do not return that data to them.

Reply:
fixed, check `toBlock` at api layer